packages/trader/src/sass/app/_common/components/contract-audit.scss

```diff
@@ -98,4 +98,37 @@ $HEIGHT: calc(100vh - 34em);
             }
         }
     }
+    &__selected-tick {
+        align-items: center;
+        background-color: var(--general-section-1);
+        border-radius: 0.4rem;
+        display: flex;
+        flex-shrink: 0;
+        height: 2.4rem;
+        justify-content: center;
+        padding: 0.4rem;
+        width: 2.4rem;
+
+        &--marker {
+            align-items: center;
+            border-radius: 50%;
+            border: 1px solid $color-black;
+            display: flex;
+            height: 1.6rem;
+            justify-content: center;
+            width: 1.6rem;
+
+            &--won {
+                background-color: $color-green-1;
```

Comment:
Can we use css variables here? var(--text-profit-success)

Reply:
Done 👍🏻